### ISSUE TITLE
The size of the box showing groups names is too small 

### ISSUE THREAD

**Issue by elhananjair** [nextcloud/announcementcenter#356]
### Steps to reproduce
1. Click on options for the announcement 
2. Click on the input field that shows group names 

### Expected behaviour
well I expected at least somehow wide container to show group names which can exceed the size of its container. 

### Actual behaviour
It fits 100% with-in its container which shows some options with check boxes, but doesn't show group names that are somehow have long names. 

### Server configuration

**Operating system**:
Fedora Server 33
**Web server:**
Apache
**Database:**
PostgreSQL
**PHP version:**
7.4.19
**Nextcloud version:** (see Nextcloud admin page)
21.0.2
**Where did you install Nextcloud from:**
Official website by downloading the .zip file 
**Signing status:**

 
![image](https://user-images.githubusercontent.com/69236638/121608224-2b76b780-ca5a-11eb-9f26-ff76dc4ecd8e.png)

**Comment by elhananjair** [comment]
@nickvergessen is the a thing that can be fixed? For long group names and with similar words its difficult to distinguish between the two.

**Comment by elhananjair** [comment]
> I'm not sure if the app can control the width of the popover

oww okay, is the generic style in NC?

**Comment by nickvergessen** [comment]
Yes, it's ready to use vue components: https://nextcloud-vue-components.netlify.app/

**Comment by elhananjair** [comment]
> Yes, it's ready to use vue components: https://nextcloud-vue-components.netlify.app/

Okay therefore let me mention this issue on Nextcloud Server repo
